components/http_foundation.rst

```diff
@@ -547,13 +547,22 @@ class, which can make this even easier::
 
     use Symfony\Component\HttpFoundation\JsonResponse;
 
+    // if you know the data to send when creating the response
+    $response = new JsonResponse(array('data' => 123));
+
+    // if you don't know the data to send when creating the response
     $response = new JsonResponse();
-    $response->setData(array(
-        'data' => 123
-    ));
+    // ...
+    $response->setData(array('data' => 123));
+
+    // if the data to send is already encoded in JSON
+    $response = JsonResponse::fromJsonString('{ "data": 123 }');
+
+.. versionadded:: 3.2
```

Comment:
Then this blog post is wrong too 😢  http://symfony.com/blog/new-in-symfony-3-2-dx-improvements#added-a-named-constructor-to-jsonresponse